test/remote/gateways/remote_redsys_test.rb

```diff
@@ -175,21 +175,16 @@ def test_failed_void
     authorize = @gateway.authorize(@amount, @credit_card, @options)
     assert_success authorize
 
-    void = @gateway.void(authorize.authorization)
-    assert_success void
-
-    another_void = @gateway.void(authorize.authorization)
-    assert_failure another_void
-    assert_equal 'SIS0222 ERROR', another_void.message
+    void = @gateway.void(authorize.authorization << '123')
+    assert_failure void
+    assert_equal 'SIS0007 ERROR', void.message
```

Comment:
I think these same changes are needed for the `sha256` version of the remote tessts as well